code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -150,9 +146,10 @@
 		if("Other Signature")
 			mode = 2
 			switch(alert("Search for item signature or DNA fragment?" , "Signature Mode Select" , "" , "Item" , "DNA"))
+
 				if("Item")
 					var/datum/objective/steal/itemlist
-					itemlist = itemlist // To supress a 'variable defined but not used' error.
+					itemlist = itemlist
 					var/targetitem = input("Select item to search for.", "Item Mode Select","") as null|anything in itemlist.possible_items

```

Comment:
Is itemlist actually initialized to anything when it's used here?

Reply:
No.  `itemlist` is in fat a global static variable that needs no init.  The self-assignment is just as the comment mentioned: a workaround for a byond bug to suppress DreamMaker issuing a spurious warning.